media-video/syncplay/syncplay-1.6.4a.ebuild

```diff
@@ -18,32 +18,30 @@ IUSE="+client +server vlc"
 REQUIRED_USE="vlc? ( client )
 	${PYTHON_REQUIRED_USE}"
 
-DEPEND=""
 # TODO: investigate the possibility of enabling PyQt5 gui
 # possible licensing concerns
 RDEPEND="${PYTHON_DEPS}
-	>=dev-python/certifi-2018.11.29
+	>=dev-python/certifi-2018.11.29[${PYTHON_USEDEP}]
```

Comment:
Squash the fixes for the issues you've introduced. Issues that were present previously are fine in a separate commit but I suppose it's easier to squash them all.

Another interesting option is to do all the changes on -9999 first, then make a bump on top of that.